egs/swc/s5b/RESULTS.txt

```diff
@@ -0,0 +1,81 @@
+%WER 77.9 | 10318 56260 | 24.9 48.4 26.7 2.9 77.9 70.9 | -23.513 | exp/sdm1/SA1/dnn4noSAT_pretrain-dbn_dnn_smbr/decode_eval_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7_it4/ascore_11/eval_o4.ctm.filt.sys
+
+
+2. With speaker adaptation
+
+- DNN
+%WER 79.5 | 10311 58361 | 24.3 52.1 23.6 3.7 79.5 72.5 | -24.385 | exp/sdm1/SA1/dnn4_pretrain-dbn_dnn/decode_dev_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7/ascore_10/dev_o4.ctm.filt.sys
+%WER 80.7 | 9919 56091 | 23.3 52.6 24.1 4.0 80.7 74.9 | -24.760 | exp/sdm1/SA1/dnn4_pretrain-dbn_dnn/decode_eval_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7/ascore_11/eval_o4.ctm.filt.sys
+
+- DNN+SMBR
+%WER 77.4 | 9959 58553 | 25.9 49.0 25.2 3.2 77.4 74.5 | -23.462 | exp/sdm1/SA1/dnn4_pretrain-dbn_dnn_smbr/decode_dev_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7_it4/ascore_11/dev_o4.ctm.filt.sys
+%WER 78.9 | 9932 56260 | 24.4 49.1 26.5 3.3 78.9 74.4 | -23.791 | exp/sdm1/SA1/dnn4_pretrain-dbn_dnn_smbr/decode_eval_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7_it4/ascore_12/eval_o4.ctm.filt.sys
+
+
+======================
+MDM: (TBL1, 8 channel, BeamformIt)
+
+1. Without speaker adaptation
+
+- DNN
+%WER 75.7 | 10492 58553 | 28.4 50.6 21.0 4.1 75.7 70.3 | -23.321 | exp/mdm8/SA1/dnn4noSAT_pretrain-dbn_dnn/decode_dev_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7/ascore_10/dev_o4.ctm.filt.sys
+%WER 77.2 | 9951 56091 | 27.7 52.7 19.6 4.9 77.2 73.9 | -23.818 | exp/mdm8/SA1/dnn4noSAT_pretrain-dbn_dnn/decode_eval_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7/ascore_10/eval_o4.ctm.filt.sys
+
+- DNN+SMBR
+%WER 72.9 | 10477 58545 | 31.1 48.1 20.8 4.0 72.9 69.9 | -22.662 | exp/mdm8/SA1/dnn4noSAT_pretrain-dbn_dnn_smbr/decode_dev_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7_it4/ascore_10/dev_o4.ctm.filt.sys
+%WER 74.6 | 9866 56091 | 29.2 47.6 23.2 3.8 74.6 73.7 | -22.781 | exp/mdm8/SA1/dnn4noSAT_pretrain-dbn_dnn_smbr/decode_eval_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7_it4/ascore_11/eval_o4.ctm.filt.sys
+
+
+2. With speaker adaptation
+
+- DNN
+%WER 75.6 | 9338 58361 | 28.7 50.2 21.1 4.3 75.6 79.5 | -23.266 | exp/mdm8/SA1/dnn4_pretrain-dbn_dnn/decode_dev_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7/ascore_10/dev_o4.ctm.filt.sys
+%WER 77.4 | 9069 56260 | 26.4 49.8 23.8 3.9 77.4 81.8 | -23.533 | exp/mdm8/SA1/dnn4_pretrain-dbn_dnn/decode_eval_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7/ascore_12/eval_o4.ctm.filt.sys
+
+- DNN+SMBR
+%WER 73.4 | 9373 58361 | 30.7 47.7 21.6 4.1 73.4 78.5 | -22.653 | exp/mdm8/SA1/dnn4_pretrain-dbn_dnn_smbr/decode_dev_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7_it4/ascore_10/dev_o4.ctm.filt.sys
+%WER 75.4 | 10621 56091 | 28.8 47.7 23.5 4.2 75.4 69.1 | -22.909 | exp/mdm8/SA1/dnn4_pretrain-dbn_dnn_smbr/decode_eval_30k-4gram-int-swc.swc.v2.30k-combilex.4g-int.arpa.pr1-7_it4/ascore_11/eval_o4.ctm.filt.sys
+
+
+====================== 
```

Comment:
@vesis84 unfortunately there are no results with nnet3. When I wanted to re-run the experiments with nnet3, I realized that I need to first solve a lot of infrastructure issues with the libraries and paths in our grid. I did not have enough time to do everything so I decided to leave nnet3 for someone else in the future. Actually I am leaving Sheffield and I will start my new job in a few days. The whole last week I was engaged in the visa application and relocation things. So I am very sorry for the very slow reaction to the comments. Thank you and Dan a lot for reviewing the codes. I will do my best to catch up in coming days.

By the way, I am not very sure why the speaker adaptation doesn't work.